internal/ingress/status/status.go

```diff
@@ -165,7 +165,12 @@ func NewStatusSyncer(config Config) Syncer {
 // ingress controller is currently running
 func (s *statusSync) runningAddresses() ([]string, error) {
 	if s.PublishStatusAddress != "" {
-		return []string{s.PublishStatusAddress}, nil
+		multipleAddrs := strings.Split(s.PublishStatusAddress, ",")
+		addresses := make([]string, 0)
+		for i := range multipleAddrs {
+			addresses = append(addresses, multipleAddrs[i])
+		}
+		return addresses, nil
```

Comment:
Why are you copying `multipleAddrs` into a new array instead of directly returning it?

Reply:
good point, fixed it